src/govuk/components/character-count/character-count.js

```diff
@@ -153,6 +166,31 @@ CharacterCount.prototype.updateCountMessage = function () {
   }
 
   // Update message
+  countMessage.innerHTML = this.formatUpdateMessage()
+}
+
+// Update screen reader-specific counter
+CharacterCount.prototype.updateScreenReaderCountMessage = function () {
+  var countMessage = this.$screenReaderCountMessage
+
+  // If other the threshold, remove the aria-hidden attribute, allowing screen
+  // readers to announce the content of the element.
+  if (this.isOverThreshold()) {
+    countMessage.removeAttribute('aria-hidden')
+  } else {
+    countMessage.setAttribute('aria-hidden', true)
+  }
+
+  // Update message
+  countMessage.innerHTML = this.formatUpdateMessage()
+}
+
+// Format update message
+CharacterCount.prototype.formatUpdateMessage = function () {
```

Comment:
Really nitpicky, but I wonder if this function name could be clearer. At the moment, it kind of sounds like we're passing it a bunch of things which it's formatting. Maybe something like `formattedUpdateMessage` is clearer?